INCommonsExample/INCommonsExample/HomeView.swift

```diff
@@ -17,7 +17,9 @@ struct HomeView: View {
 		ExampleViewFactory(title: "Overlay Button", view: { AnyView(OverlayButtonExample()) }),
 		ExampleViewFactory(title: "Share Sheet", view: { AnyView(ShareSheet()) }),
 		ExampleViewFactory(title: "View Condition", view: { AnyView(ViewConditions()) }),
-		ExampleViewFactory(title: "View Frame", view: { AnyView(ViewFrame()) })
+		ExampleViewFactory(title: "View Frame", view: { AnyView(ViewFrame()) }),
+        ExampleViewFactory(title: "Color Picker", view: { AnyView(ColorPickerExample()) })
+
```

Comment:
Please format this appropriately
```suggestion
		ExampleViewFactory(title: "View Frame", view: { AnyView(ViewFrame()) }),
		ExampleViewFactory(title: "Color Picker", view: { AnyView(ColorPickerExample()) })
```